scripts/globals/trusts.lua

```diff
@@ -0,0 +1,156 @@
+require("scripts/globals/status")
+require("scripts/globals/msg")
+
+-----------------------------------
+--
+--  TRUST ID
+--
+-----------------------------------
+dsp = dsp or {}
+dsp.trust = dsp.trust or {}
+
+-----------------------------------
+--  Trust
+-----------------------------------
+dsp.trust =
+{
+	SHANTOTTO           =  896;
```

Comment:
NO NO NO a thousand times NO. if you are enumerating every single ID you are doing it wrong. This is why this should have been PR'd before you tried to do ALL of them. Because you went about it wrong.

Reply:
its an easy fix